shachain/element_test.go

```diff
@@ -0,0 +1,179 @@
+package shachain
+
+import (
+	"reflect"
+	"testing"
+)
+
+// bitsToUint64 is a helper function which takes 'n' last bits as input and
+// create uint64 value.
+// Example:
+//  Input: 0,1,1,0,0
+//  Output: 0b000000000000000000000000000000000000000[01100] == 12
+func bitsToUint64(bs ...uint64) uint64 {
+	if len(bs) > 64 {
+		panic("number of elements should be lower then 64")
```

Comment:
If function will return an error, than I will not be able use it to initialise the `deriveIndexTests`, so should I just remove the checks?